packages/astro/src/core/errors/errors-data.ts

```diff
@@ -890,8 +884,7 @@ export const MissingSharp = {
 	name: 'MissingSharp',
 	title: 'Could not find Sharp.',
 	message:
-		'Could not find Sharp. Please install Sharp (`sharp`) manually into your project or migrate to another image service.',
-	hint: "See Sharp's installation instructions for more information: https://sharp.pixelplumbing.com/install. If you are not relying on `astro:assets` to optimize, transform, or process any images, you can configure a passthrough image service instead of installing Sharp. See https://docs.astro.build/en/reference/errors/missing-sharp for more information.\n\nSee https://docs.astro.build/en/guides/images/#default-image-service for more information on how to migrate to another image service.",
+		'Could not find npm dependency package "sharp". Please install Sharp manually into your project or migrate to another image service.',
```

Comment:
This loses the helpful context about a passthrough/noop service being available, which is not fully conveyed by "install or migrate to another service." (I wouldn't get from this that a non-service would exist.) Could still be done briefly, maybe something like:

```suggestion
		'Could not find npm dependency package "sharp". Please install Sharp manually into your project or choose another image service configuration.',
```